src/bcf/record.rs

```diff
@@ -558,6 +657,37 @@ impl Record {
         })
     }
 
+    /// Retrieve data for a `FORMAT` field
+    ///
+    /// # Example
+    /// *Note: some boilerplate for the example is hidden for clarity. See [module documentation](../index.html#example-writing)
+    /// for an example of the setup used here.*
+    ///
+    /// ```rust
+    /// # use rust_htslib::bcf::{Format, Writer};
+    /// # use rust_htslib::bcf::header::Header;
+    /// #
+    /// # // Create minimal VCF header with a single sample
+    /// # let mut header = Header::new();
+    /// header.push_sample(b"sample1").push_sample(b"sample2").push_record(br#"##FORMAT=<ID=DP,Number=1,Type=Integer,Description="Read Depth">"#);
+    /// #
+    /// # // Write uncompressed VCF to stdout with above header and get an empty record
+    /// # let mut vcf = Writer::from_stdout(&header, true, Format::VCF).unwrap();
+    /// # let mut record = vcf.empty_record();
```

Comment:
Maybe make this line visible in the docs? So that it is clear that `record` is an `empty_record()` to start with.
```suggestion
/// let mut record = vcf.empty_record();
```